scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/simulation/IncrementalSimulationTest.java

```diff
@@ -36,11 +37,11 @@ public void init() throws TaskSpecType.UnconstructableTaskSpecException, Invalid
   @Test
   public void simulationResultsTest(){
     //ensures that simulation results are generated until the end of the last act;
-    var simResults = incrementalSimulationDriver.getSimulationResults();
+    var simResults = incrementalSimulationDriver.getSimulationResults(Instant.now());
     assert(simResults.realProfiles.get("/utcClock").getRight().get(0).getLeft().isEqualTo(endOfLastAct));
     /*ensures that when current simulation results cover more than the asked period and that nothing has happened
     between two requests, the same results are returned*/
-    var simResults2 = incrementalSimulationDriver.getSimulationResultsUpTo(Duration.of(7,SECONDS));
+    var simResults2 = incrementalSimulationDriver.getSimulationResultsUpTo(Instant.now(), Duration.of(7, SECONDS));
```

Comment:
[Clarification:] Does `simResults2` return the same object in `simResults` because its duration is a subset of `simResults`'s?

Reply:
Absolutely, it is not recomputed because of what you mentioned.